onboard-app-guilherme/src/validations/login-validations.ts

```diff
@@ -0,0 +1,42 @@
+import { emailRegex, hasDigitRegex, hasLetterRegex } from "../utils/regex";
+
+export type LoginErrors = {
+  email?: string;
+  password?: string;
+};
```

Comment:
```suggestion
export interface LoginErrors {
  email?: string;
  password?: string;
};
```

Nós usamos interface como convenção. Não é tão diferente de `type`, mas é mais flexível, permite que usemos `extends`.